### PR TITLE
#1416 Company Information validation

### PR DESCRIPTION
**Pull request by desfero**
#1416

### CODE REVIEW

File: app/components/shared/forms/formField/FormError.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { isNonValid } from "./utils";
+
+import * as styles from "./FormStyles.module.scss";
+
+export interface IProps {
+  name: string;
+  defaultMessage?: string;
+}
+
+const FormError: React.SFC<IProps> = ({ name, defaultMessage }) => (
```
Comment (desfero) [inline app/components/shared/forms/formField/FormError.tsx]: For now `FormError` is used only inside a couple of fields. https://github.com/Neufund/platform-frontend/issues/1484

File: app/components/shared/forms/index.tsx [hunk 2]
```diff
@@ -1,13 +1,14 @@
-export * from "./formField/FormField";
+export * from "./formField/FormCategoryDistribution";
```
Comment (desfero) [inline app/components/shared/forms/index.tsx]: Sorted alphabetically + reexport `FormError`

**Comment by rudolfix** [comment]
I'm afraid that e2e test for eto forms will not pass - it checks if forms are fully filled and counter logic changed? @desfero

**Comment by desfero** [comment]
@rudolfix strange, after merging with latest master e2e tests pass

**Comment by rudolfix** [comment]
and now it doesn't @desfero seems a required field is not required anymore... do you want me to fix it?

**Comment by rudolfix** [comment]
```
 Eto Company Information Field Validation
    1) should correctly validate required fields


  0 passing (12s)
  1 failing

  1) Eto Company Information Field Validation should correctly validate required fields:

      AssertionError: expected '' to equal 'This field is required'
      + expected - actual

      +This field is required
      
      at getRet (https://localhost:9090/__cypress/runner/cypress_runner.js:56968:16)
      at tryCatcher (https://localhost:9090/__cypress/runner/cypress_runner.js:7091:23)
      at Function.Promise.attempt.Promise.try (https://localhost:9090/__cypress/runner/cypress_runner.js:4470:29)
      at Context.thenFn (https://localhost:9090/__cypress/runner/cypress_runner.js:56982:26)
      at Context.then (https://localhost:9090/__cypress/runner/cypress_runner.js:57244:21)
      at Context.<anonymous> (https://localhost:9090/__cypress/runner/cypress_runner.js:63977:21)
      at https://localhost:9090/__cypress/runner/cypress_runner.js:63686:33
      at tryCatcher (https://localhost:9090/__cypress/runner/cypress_runner.js:7091:23)
      at Promise._settlePromiseFromHandler (https://localhost:9090/__cypress/runner/cypress_runner.js:5113:31)
      at Promise._settlePromise (https://localhost:9090/__cypress/runner/cypress_runner.js:5170:18)
      at Promise._settlePromiseCtx (https://localhost:9090/__cypress/runner/cypress_runner.js:5207:10)
      at Async._drainQueue (https://localhost:9090/__cypress/runner/cypress_runner.js:2028:12)
      at Async._drainQueues (https://localhost:9090/__cypress/runner/cypress_runner.js:2033:10)
      at Async.drainQueues (https://localhost:9090/__cypress/runner/cypress_runner.js:1907:14)
      at <anonymous>
```

**Comment by desfero** [comment]
@rudolfix ahhh, we merge your pr that change the optionality of fields, let me fix it :D

**Pull request merged**